plugins/carbon_black_response/requirements.txt

```diff
@@ -1,4 +1,8 @@
 # List third-party dependencies here, separated by newlines. 
 # All dependencies must be version-pinned, eg. requests==1.2.0
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
-cbapi==1.6.2
+cbapi==1.7.10
+validators==0.21.0
+urllib3==2.2.2
+requests==2.32.2
+certifi==2024.7.4
```

Comment:
why are all the modules being pinned?

Reply:
To address Snyk vulnerabilities. When running snyk in terminal to test, these were the required modules to be included